### ISSUE TITLE
can't run py2dsc-deb on 3.17.1 

### ISSUE THREAD

**Issue by miekg** [ietf-tools/xml2rfc#982]
### Describe the issue

I'm using the above command to create a debian package and thus far it works perfectly, but with the 3.17.1 release I get:

~~~
-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
running the following command in directory: deb_dist/tmp_py2dsc/xml2rfc-3.17.1
/usr/bin/python3 setup.py --command-packages stdeb.command sdist_dsc --dist-dir=/tmp/tmp.3JIAJu8QH5/deb_dist --use-premade-distfile=/tmp/tmp.3JIAJu8QH5/xml2rfc-3.17.1.tar.gz
-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/pkg_resources/_vendor/packaging/requirements.py", line 102, in __init__
    req = REQUIREMENT.parseString(requirement_string)
  File "/usr/lib/python3/dist-packages/pkg_resources/_vendor/pyparsing.py", line 1654, in parseString
    raise exc
  File "/usr/lib/python3/dist-packages/pkg_resources/_vendor/pyparsing.py", line 1644, in parseString
    loc, tokens = self._parse( instring, 0 )
  File "/usr/lib/python3/dist-packages/pkg_resources/_vendor/pyparsing.py", line 1402, in _parseNoCache
    loc,tokens = self.parseImpl( instring, preloc, doActions )
  File "/usr/lib/python3/dist-packages/pkg_resources/_vendor/pyparsing.py", line 3417, in parseImpl
    loc, exprtokens = e._parse( instring, loc, doActions )
  File "/usr/lib/python3/dist-packages/pkg_resources/_vendor/pyparsing.py", line 1406, in _parseNoCache
    loc,tokens = self.parseImpl( instring, preloc, doActions )
  File "/usr/lib/python3/dist-packages/pkg_resources/_vendor/pyparsing.py", line 3205, in parseImpl
    raise ParseException(instring, loc, self.errmsg, self)
pkg_resources._vendor.pyparsing.ParseException: Expected stringEnd (at char 4), (line:1, col:5)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/tmp.3JIAJu8QH5/deb_dist/tmp_py2dsc/xml2rfc-3.17.1/setup.py", line 3, in <module>
    setup(
  File "/usr/lib/python3/dist-packages/setuptools/__init__.py", line 152, in setup
    _install_setup_requires(attrs)
  File "/usr/lib/python3/dist-packages/setuptools/__init__.py", line 145, in _install_setup_requires
    dist.parse_config_files(ignore_option_errors=True)
  File "/usr/lib/python3/dist-packages/setuptools/dist.py", line 807, in parse_config_files
    self._finalize_requires()
  File "/usr/lib/python3/dist-packages/setuptools/dist.py", line 534, in _finalize_requires
    self._move_install_requirements_markers()
  File "/usr/lib/python3/dist-packages/setuptools/dist.py", line 573, in _move_install_requirements_markers
    inst_reqs = list(pkg_resources.parse_requirements(spec_inst_reqs))
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 3099, in parse_requirements
    yield Requirement(line)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 3109, in __init__
    super(Requirement, self).__init__(requirement_string)
  File "/usr/lib/python3/dist-packages/pkg_resources/_vendor/packaging/requirements.py", line 104, in __init__
    raise InvalidRequirement(
pkg_resources.extern.packaging.requirements.InvalidRequirement: Parse error at "': requir'": Expected stringEnd
ERROR running: /usr/bin/python3 setup.py --command-packages stdeb.command sdist_dsc --dist-dir=/tmp/tmp.3JIAJu8QH5/deb_dist --use-premade-distfile=/tmp/tmp.3JIAJu8QH5/xml2rfc-3.17.1.tar.gz
ERROR in deb_dist/tmp_py2dsc/xml2rfc-3.17.1
~~~

Running `python3 setup.py` gives the same error. Looking at `setup.py` it's quite a bit shorter than the one in 3.16.x, but I don't know enough about Python to say this matters.

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by miekg** [comment]
Ah, see 2b112e21474c77f84c492e1c399e7d9cb4e480b8 changed all this, let me look if py2dsc-deb understands this too

**Comment by kesara** [comment]
I think this issue was fixed with #1057.
Please reopen if you still get the same error.
